cmake/std/atdm/waterman/tweaks/CUDA_COMMON_TWEAKS.cmake

```diff
@@ -1,6 +1,3 @@
 # This test fails consistently (#2751)
 ATDM_SET_ENABLE(PanzerAdaptersSTK_MixedPoissonExample-ConvTest-Hex-Order-3_DISABLE ON)
 
-# This test runs out of CUDA memory all on its own (#3340)
```

Comment:
@hkthorn, this commit should not be in this PR.   Please rebase your branch on top of current 'develop' and force push your branch and and hopefully this commit will go away.